src/App/Action/Account/Meetup/CancelCheckInAction.php

```diff
@@ -56,6 +56,6 @@ public function __invoke(Request $request, Response $response, callable $next =
             $meetup->cancelCheckIn($user);
         });
 
-        return new RedirectResponse($this->urlHelper->generate('account-meetup-view', ['uuid' => $meetup->getId()]));
+        return new RedirectResponse($this->urlHelper->generate('account-meetup-view', ['uuid' => $meetup->getId()]).'#attendees');
```

Comment:
Fourth param of the URL helper is the URI fragment: https://zend-expressive.readthedocs.io/en/latest/features/helpers/url-helper/

:slightly_smiling_face:

Reply:
I'll take a look and fix it. Thanks :)

Reply:
@jezemery I think changing this will fix the tests too :slightly_smiling_face:

Reply:
@lboynton the version of expressive we're using doesn't support this - https://github.com/zendframework/zend-expressive/blob/1.0.0/doc/book/features/helpers/url-helper.md 

Ideas?

Reply:
@jezemery Despair?

I guess in that case do what you've done and append '#attendees' to the line here: 

- https://github.com/phphants/phph-site/blob/256c31eac92919c95065367b9044a23e3f2e5126/test/AppTest/Action/Account/Meetup/CancelCheckInActionTest.php#L54
- https://github.com/phphants/phph-site/blob/256c31eac92919c95065367b9044a23e3f2e5126/test/AppTest/Action/Account/Meetup/CheckInUserActionTest.php#L53

Reply:
Aren't we using version 2 though? https://github.com/phphants/phph-site/blob/master/composer.json#L12

Reply:
You're not wrong, but when I go and find that function, there's no mention of features available in v2.0 😕

Reply:
Ah, it was added in version 3 of zend-expressive-helpers: https://github.com/zendframework/zend-expressive-helpers/blob/develop/CHANGELOG.md#300---2016-01-11

Looking at the changelog, we should be good to upgrade I think.

Reply:
That's weird... it's in the documentation for v2.0...